src/tasks/pollYoutube.js

```diff
@@ -0,0 +1,27 @@
+const { red, green } = require('colors/safe');
+const { schedule } = require('node-cron');
+const { fetchLatestYoutubeVideos } = require('../helpers/fetchData');
+const Video = require('../api/video/video.model');
+
+schedule('59 * * * *', async () => {
+  try {
+    let { date: lastDate } = await Video.findOne({}).sort({ date: -1 });
```

Comment:
I am second guessing this logic now, If this is date based we may run into an issue if more than one video is posted in one day.

Reply:
you´re right!
In the if statement i should check if the video id is the same too.

Reply:
Should be fine, since date includes hour and minute.

Reply:
Yesterday cj upload two new vids, the cronjob detected and added both. In the next cronjob (1 hour later ) no vid was detected. So i think it should work.

But no problem to add the extra if statement just for being sure

Reply:
If it's not broken, there is no point fixing it